mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule BlockBox.MixProject do
   def project do
     [
       app: :blockbox,
-      version: "1.1.0",
+      version: "1.1.1",
```

Comment:
this isn't the only place where you have to change the version. It should also be updated in the BlockBox moduledoc and the README

Reply:
I'll make this change